packages/component-base/test/virtualizer-unlimited-size.test.js

```diff
@@ -22,6 +22,13 @@ describe('unlimited size', () => {
         el.index = index;
         el.id = `item-${index}`;
         el.textContent = el.id;
+        el.style.right = '0';
+        el.style.left = '0';
+        el.style.display = 'flex';
+        el.style.alignItems = 'center';
+        el.style.background = index % 2 === 0 ? '#e7e7e7' : '#d0d0d0';
+        el.style.height = '30px';
```

Comment:
**note:** Setting fixed height prevents test failures related to differences in font rendering across browsers.